nav_msgs/Path superior
nav_msgs/Path inferior
---
std_msgs/Float64MultiArray collisionTimes